### PR TITLE
Yo 43

### PR DESCRIPTION
**Pull request by ladinu**
# Description

Added `pickupLocation` field to `Carpool`.

### CODE REVIEW

File: integration/controllers/carpool.ctrl.spec.ts [hunk 1]
```diff
@@ -322,6 +331,7 @@ describe('Carpool controller', () => {
     var carpoolUpdate:CarpoolSvc.CarpoolUpdateModel = {
       'name': 'name updated',
       'description': 'description updated'
+      // TODO: Add pickupLocation
```
Comment (ianaldrighetti) [inline integration/controllers/carpool.ctrl.spec.ts:334]: This supposed to be here?

File: integration/controllers/carpool.ctrl.spec.ts [hunk 2]
```diff
@@ -331,6 +341,7 @@ describe('Carpool controller', () => {
         .then((updatedCarpool) => {
           expect(updatedCarpool['name']).toEqual(carpoolUpdate.name);
           expect(updatedCarpool['description']).toEqual(carpoolUpdate.description);
+      // TODO: Add pickupLocation
```
Comment (ianaldrighetti) [inline integration/controllers/carpool.ctrl.spec.ts:344]: This too?

**Comment by ianaldrighetti** [comment]
#peer

File: integration/controllers/carpool.ctrl.spec.ts (hunk 1 above)
Comment (ladinu) [inline integration/controllers/carpool.ctrl.spec.ts:334]: Yeah, those are for @bbuxell :smile:

**Comment by bbuxell** [comment]
#peer

**Pull request merged**